### PR TITLE
[MM-19802] Show associated project in select subscription modal

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

This PR introduces a column in the select subscription modal, for showing the project name for a given subscription.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-19802

#### Screenshots

image

### CODE REVIEW

File: webapp/src/components/modals/channel_settings/select_channel_subscription.tsx [hunk 1]
```diff
@@ -48,6 +48,47 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
...
+            >
+                <td>
+                    <span>{sub.name || '(no name)'}</span>
+                </td>
+                <td>
+                    <span>{projectName}</span>
+                </td>
+                <td>
+                    <button
+                        className='style--none color--link'
```
Comment (jfrerich) [inline webapp/src/components/modals/channel_settings/select_channel_subscription.tsx]:
```suggestion
style={{marginLeft: '0px'}}
```

File: webapp/src/components/modals/channel_settings/select_channel_subscription.tsx [hunk 2]
```diff
@@ -48,6 +48,47 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
...
+                        onClick={(): void => this.props.showEditChannelSubscription(sub)}
```
Comment (jfrerich) [inline webapp/src/components/modals/channel_settings/select_channel_subscription.tsx]:
```suggestion
                        style={{marginLeft: '0px'}}
                        onClick={(): void => this.props.showEditChannelSubscription(sub)}
```

File: webapp/src/components/modals/channel_settings/select_channel_subscription.tsx (hunk 1 above)
Comment (jfrerich) [inline webapp/src/components/modals/channel_settings/select_channel_subscription.tsx]:
```suggestion
                        className='style--none color--link'
                        style={{marginLeft: '0px'}}
```

File: webapp/src/components/modals/channel_settings/select_channel_subscription.tsx [hunk 3]
```diff
@@ -48,6 +48,47 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
...
+                        type='button'
+                    >
+                        {'Edit'}
+                    </button>
+                    {' - '}
+                    <button
+                        className='style--none color--link'
```
Comment (jfrerich) [inline webapp/src/components/modals/channel_settings/select_channel_subscription.tsx]:
```suggestion
                        className='style--none color--link'
                        style={{marginLeft: '0px'}}
```

File: webapp/src/components/modals/channel_settings/select_channel_subscription.tsx (hunk 1 above)
Comment (jfrerich) [reply webapp/src/components/modals/channel_settings/select_channel_subscription.tsx]: @mickmister Here's a suggestion.  The buttons are styled with `margin-left: 15px`, which is creating two problems. 1) The shift of the `Edit - Delete` text to the right of the `Actions` header and 2) the uneven centering of the dash between Edit and Delete.

**Comment by mickmister** [comment]
@jfrerich styling issue addressed

image

**Comment by jfrerich** [comment]
@mickmister Looks great! Approving..

**Pull request merged**